pkg/controller/plan/kubevirt.go

```diff
@@ -1237,8 +1237,28 @@ func (r *KubeVirt) guestConversionPod(vm *plan.VMStatus, vmVolumes []cnv.Volume,
 	allowPrivilageEscalation := false
 	// virt-v2v image
 	var virtV2vImage string
+	var initContainers []core.Container
 	if r.Context.UseEl9VirtV2v() {
 		virtV2vImage = Settings.Migration.VirtV2vImageCold
+		if vddkImage, found := r.Source.Provider.Spec.Settings["vddkInitImage"]; found {
+			initContainers = append(initContainers, core.Container{
+						Name:            "vddk-side-car",
+						Image:           vddkImage,
+						ImagePullPolicy: core.PullIfNotPresent,
+						VolumeMounts: []core.VolumeMount{
+							{
+								Name:      "vddk-vol-mount",
+								MountPath: "/opt",
+							},
+						},
+						SecurityContext: &core.SecurityContext{
+							AllowPrivilegeEscalation: &allowPrivilageEscalation,
+							Capabilities: &core.Capabilities{
+								Drop: []core.Capability{"ALL"},
+							},
+						},
+					})
+		}
```

Comment:
need to add the vddk side-container in the else-block below, right?

Reply:
Actually this is wrong. It does not belong into this scope.